pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -2087,4 +2098,318 @@ var _ = Describe("v1beta1 inference service controller", func() {
+			// Create configmap
+			copiedConfigs := make(map[string]string)
+			for key, value := range configs {
+				if key == "storageInitializer" {
+					copiedConfigs[key] = `{
+							"image" : "kserve/storage-initializer:latest",
+							"memoryRequest": "100Mi",
+							"memoryLimit": "1Gi",
+							"cpuRequest": "100m",
+							"cpuLimit": "1",
+							"CaBundleConfigMapName": "test-cabundle-with-wrong-file-name",
+							"caBundleVolumeMountPath": "/etc/ssl/custom-certs",
+							"enableDirectPvcVolumeMount": false						
+					}`
+				} else {
+					copiedConfigs[key] = value
+				}
+			}
+			var configMap = &v1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      constants.InferenceServiceConfigMapName,
+					Namespace: constants.KServeNamespace,
+				},
+				Data: copiedConfigs,
+			}
+
+			Expect(k8sClient.Create(context.TODO(), configMap)).NotTo(HaveOccurred())
+			defer k8sClient.Delete(context.TODO(), configMap)
+
+			// Create original cabundle configmap with wrong file name
+			cabundleConfigMapData := make(map[string]string)
+			cabundleConfigMapData["wrong-cabundle-name.crt"] = "SAMPLE_CA_BUNDLE"
+			var originalCabundleConfigMap = &v1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-cabundle-with-wrong-file-name",
+					Namespace: constants.KServeNamespace,
+				},
+				Data: cabundleConfigMapData,
+			}
+
+			Expect(k8sClient.Create(context.TODO(), originalCabundleConfigMap)).NotTo(HaveOccurred())
+			defer k8sClient.Delete(context.TODO(), originalCabundleConfigMap)
+
+			By("By creating a new InferenceService")
+			serviceName := "sample-isvc-3"
+			var expectedRequest = reconcile.Request{NamespacedName: types.NamespacedName{Name: serviceName, Namespace: "default"}}
+			var serviceKey = expectedRequest.NamespacedName
+			var storageUri = "s3://test/mnist/export"
+			ctx := context.Background()
+			isvc := &v1beta1.InferenceService{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      serviceKey.Name,
+					Namespace: serviceKey.Namespace,
+				},
+				Spec: v1beta1.InferenceServiceSpec{
+					Predictor: v1beta1.PredictorSpec{
+						ComponentExtensionSpec: v1beta1.ComponentExtensionSpec{
+							MinReplicas: v1beta1.GetIntReference(1),
+							MaxReplicas: 3,
+						},
+						Tensorflow: &v1beta1.TFServingSpec{
+							PredictorExtensionSpec: v1beta1.PredictorExtensionSpec{
+								StorageURI:     &storageUri,
+								RuntimeVersion: proto.String("1.14.0"),
+								Container: v1.Container{
+									Name:      constants.InferenceServiceContainerName,
+									Resources: defaultResource,
+								},
+							},
+						},
+					},
+				},
+			}
+			Expect(k8sClient.Create(ctx, isvc)).Should(Succeed())
+			defer k8sClient.Delete(ctx, isvc)
+
+			caBundleConfigMap := &v1.ConfigMap{}
+			Eventually(func() bool {
```

Comment:
```suggestion
			Consistently(func() bool {
```

Reply:
I think `Eventually` is fit in this place based on this definition.
```
 Eventually expects the condition to be met at some point in the future, while Consistently expects the condition to be consistently met over time. 
```
It does not matter when it can get the configmap. what we are looking for is at some point, it should be created and be readable.  WDYT?

Reply:
oh right.  line #2403 should be `Eventually` but this one can be `Consistently`